CHANGELOG.md

```diff
@@ -56,6 +56,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
    | Windows | 1.14       | amd64        |
    | Windows | 1.15       | 386          |
    | Windows | 1.14       | 386          |
+- Added `WithDefaultSampler` and `WithSpanLimits` to tracer provider. (#1633)
```

Comment:
It seems it is the wrong place to put this changelog.

Reply:
Sorry to not have checked carefully.

Reply:
That is fine. This should be handled by an automatic tool, which lowers everyone's mental burden. ;)